.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+require( '@automattic/eslint-plugin-wpvip/init' );
+
+module.exports = {
+	extends: [ 'plugin:@automattic/wpvip/recommended' ],
+	root: true,
+	env: {
+		jest: true,
+	},
+	rules: {
+		"no-prototype-builtins": 0,
```

Comment:
@ingeniumed Do you think any of these rules are more important than the others and we can apply them now? Or maybe we can make issues to address the more important exceptions (`no-eval`) soon. I get ignoring huge refactorings like `valid-jsdoc`, but it'd be great if low-hanging code quality and safety rules would be applied.

Reply:
I wanted to avoid adding more scope to this PR by resolving those, that's really why I avoided adding it in here. I also think as we build more features into this plugin, we are likely to change/upgrade that code (which might happen anyways thanks to the package modernizations)